lib/rules/prefer-object-spread.js

```diff
@@ -212,15 +212,19 @@ function defineFixer(node, sourceCode) {
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Since fixing errors reported by this rule would require changing the statement, should this be a "suggestion"?